core/src/main/java/com/linecorp/armeria/server/encoding/HttpEncoders.java

```diff
@@ -41,6 +41,11 @@ final class HttpEncoders {
 
     private static final Encoder.Parameters BROTLI_PARAMETERS = new Encoder.Parameters().setQuality(4);
 
+    static {
+        // Invoke to load Brotli native binary.
+        Brotli.isAvailable();
```

Comment:
It would be a defect of brotli4j that a Brotli native library is not initialized when `BrotliOutputStream` is created.
`Brotli.isAvailable()` is called before brotli codec is applied in the request call path. 

However, when I tried to test brotli codec with `HttpEncoders`, the test failed because Brotli native library wasn't loaded. I added this for the sake of convenience of testing.

Reply:
> However, when I tried to test brotli codec with HttpEncoders, the test failed because Brotli native library wasn't loaded. I added this for the sake of the convenience of testing.

That's interesting. Which test failed without this?

Reply:
You can see the brotli initialization error in `HttpEncodedResponseTest.shouldReleaseEncodedStreamOnError()` after removing `Brotli.isAvailable()`.

Reply:
Ah ha, I get it. `determineEncoding` which calls `Brotli.isAvailable()` is called ahead so it was no big deal when the code is used in production. 😉